content/api/entity-reference/gift/index.html

```diff
@@ -123,12 +123,12 @@
             <tr>
                 <td>gift_aid_amount</td>
                 <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of tax reclaimed from Gift Aid for this gift. For UK only.</td>
+                <td>The amount of tax reclaimed from Gift Aid for this gift.  For UK formatted databases only. This is a computed field based on total qualified gift aid amount across all <code>gift_splits</code>.</td>
```

Comment:
the gift split gift aid is actually not calculated, only the aggregated gift aid information is